settings.yml

```diff
@@ -1,13 +1,18 @@
-display_categories:
+discovery_mode:
+  type: string
```

Comment:
This would be better as an enum  theme setting, so it has a dropdown with three options